README.md

```diff
@@ -6,28 +6,46 @@ If you want to contribute join us on the [Pebble Dev Discord server](http://disc
 
 Backend/API layer requires `git`, `go`, `npm`, and `apib2swagger`.
 
-## Dev Environment Setup
-Pull down the project within your `$GOPATH`'s src folder ($GOPATH is an
-environment variable and is typically set to $HOME/go/ on \*nix). This can be
-done by running (for example) the following set of commands:
+To make the backend do anything, you also need to download a copy of the Pebble App Store. You can already start downloading it [here](https://drive.google.com/file/d/0B1rumprSXUAhTjB1aU9GUFVPUW8/view) while you setup the development environment.
 
-		GOPATH=go/
-		mkdir -p $GOPATH/src/pebble-dev
-		git clone https://github.com/pebble-dev/rebblestore-api.git $GOPATH/src/pebble-dev/rebblestore-api
+## Dev Environment Setup
+Pull down the project within your `$GOPATH`'s src folder ($GOPATH is an environment variable and is typically set to $HOME/go/ on \*nix). This can be done by running (for example) the following set of commands:
 
-Please [go fmt your code](https://blog.golang.org/go-fmt-your-code) and run `go
-test` before committing your changes. Some editor plugins (such as vim-go)
-should be able to do this automatically before save.
+        GOPATH=go/
```

Comment:
I changed it to `# export GOPATH=~/go/ # Optional if your didn't move your ~/go directory `, because if the user doesn't want it in ~/go, it's not obvious what to do.